README.md

```
@@ -35,26 +45,59 @@
+| **게시글 상세 조회** | 게시글 ID 입력 후 단일 게시글 조회                           |
+| **게시글 수정**      | 게시글 아이디 입력 후 제목과 내용을 수정제목과 내용을 입력 받아 로그인 사용자의 게시글 저장 |
+| **게시글 삭제**      | 게시글 아이디 입력 후 해당 게시글 삭제모든 게시글 리스트 조회 |
 
 ## 5. 설계
-### 5.1 클래스 설계
-### 5.2 메소드 설계
-### 5.3 데이터 설계
+
+### 5.1 Runner
+<div align="center">
+  <img src="./github_src/runner.png">
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)